ro_help/hub/templatetags/custom_tags.py

```diff
@@ -0,0 +1,20 @@
+import os
+from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
+from django.utils.safestring import mark_safe
+from django import template
+
+register = template.Library()
+
+@register.inclusion_tag('meta.html', takes_context=True)
+def meta_tags(context):
+    request = context['request']
+    image_path = os.path.join(settings.STATIC_URL, context.get('image', "images/logo-meta.png"))

```

Comment:
nit: From python 3.4 they introduced a new path library in stdlib called `path`. Maybe we can use it https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.joinpath

Reply:
Good to know, but actually this pointed to me that I confused the path with url. I changed it to urljoin as i was really trying to build an URL not a path. Thanks